### DISCUSSION TITLE
Git LFS Help

### DISCUSSION THREAD

**Discussion by tendiesman** [vercel/community#1214] [Help] [answered]
### Summary

I referred to the previous LFS thread to help solve this issue. I tried the install command
`Hey this override
set -x && git remote add origin https://github.com/tendiesman/gm_cafe.git && amazon-linux-extras install -y epel && yum install -y git-lfs && git lfs install && git lfs pull && yarn install`
But it didn't work for me. The repo is public and the domain is gm.cafe if you want to check
Would greatly appreciate any insights, thank you in advance

**Comment by tendiesman** [comment, marked as answer]
Nvm, for future reference, make sure you have pushed any updates to .gitattributes

**Reply by amyegan** [reply]
Thanks for coming back to share this solution! 😄

**Reply by amyegan** [reply]
We now have an easier way to use [Git LFS](https://vercel.com/docs/concepts/projects/overview#git-large-file-storage-lfs) with Vercel: https://vercel.com/changelog/git-lfs-support 🥳
